Documentation/migration-scripts/python-scripts/push_kafka_messages.py

```
@@ -0,0 +1,298 @@
+                "publishedAt": today,
+                "dashboardData": {
+                    "targetedProgram": linked_to,
+                    "targetedSolution": entity_id
+                }
+            }
+
+            try:
+                producer.send(TOPIC_1, value=project_json)
+                log(f"✅ Project event pushed to Kafka: {random_id}")
+                log(f"🚀 Event pushed for solution: {entity_id} -> program: {linked_to}")
+            except Exception as k_err:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)